velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -323,5 +323,36 @@ VELOX_INSTANTIATE_TEST_SUITE_P(
     LeadLagTest,
     ::testing::Values("lead"));
 
+TEST_F(LeadLagTest, int32RowsFrameBound) {
```

Comment:
The change seems generic, but the test applies only to lead/lag? Any particular reason for that?

It would be nice to add Fuzzer support for window functions to allow for this type of testing to apply more broadly.

Reply:
> Although that doesn't really need frames.

Since these functions should not be invoked with frames, let's not add that test here. It creates an impression that this is a valid use case when it is not.

Reply:
Sounds good. Thanks !